lib/msal-browser/docs/performance.md

```diff
@@ -0,0 +1,29 @@
+# Performance
```

Comment:
Can we add this doc to the README.md Advanced Topics section?

Reply:
Good call. Done.